backend/composer/services/cs_ingestion/neurondm_script.py

```diff
@@ -171,31 +268,23 @@ def main(local=False, anatomical_entities=False, anatent_simple=False):
         [g.add((s, rdfs.label, o)) for s, o in ori.graph[:rdfs.label:]]
 
     config.load_existing(g)
-    neurons = config.neurons()  # scigraph required here if deps not removed above
-
-    # ingest to composer starts here
-    mvp_ingest = [n for n in neurons if not multi_orig_dest(n)]
-
-    dims = set(p for n in neurons for p in n.edges)  # for reference
-    fcs = [for_composer(n) for n in mvp_ingest]
-    _fcne = [for_composer(n, cull=True) for n in mvp_ingest]  # exclude empties for easier manual review
-
-    # example neuron
-    n = mvp_ingest[0]
-    fc = for_composer(n)
-
-    if anatomical_entities:
-        location_summary(neurons, _noloc_query_services, anatent_simple)
-
-
-    myFile = open('./composer/services/cs_ingestion/neurons.csv', 'w')
-    writer = csv.DictWriter(myFile, fieldnames=['id','label', 'origin','dest', 'path', 'species', 'sex', 'circuit_type', 'phenotype', 'anatomical_system',  'dont_know_fcrp', 'other_phenotype','forward_connection', 'provenance', 'sentence_number', 'note_alert', '_ignore'])
-    writer.writeheader()
-    writer.writerows(fcs)
-    myFile.close()
-
-   # breakpoint()
-    return fcs
+    neurons = config.neurons()
+
+    fcs = [for_composer(n) for n in neurons]
+    composer_statements = [item for item in fcs if item is not None]
+
+    # TODO: Confirm if this can be deleted
```

Comment:
Previously we were just getting primitives (strings) in the dict produced from the neurondm data. I changed it to use some complex objects to facilitate the ingestion command, due to that I'm not sure about what we want to show in the csv that was being produced, for those objects. I commented out the csv creation for now but I would like confirmation if we don't need this. If we need it, then what would you like it to show?

Reply:
@afonsobspinto imo we can remove the csv creation. I'm not aware of anything using this csv, @ddelpiano do you?

Reply:
I kept it per @ddelpiano request but it will only show the uri and label of what was successfully ingested